t/t0013-config-file.t

```diff
@@ -121,19 +123,14 @@ test_expect_success 'start size=2 instance with ipc://' '
 	    { host="fake1" }
 	]
 	EOT
-	cat <<-EOF >attrdump.sh &&
-	#!/bin/sh
-	flux getattr size
-	flux getattr config.hostlist
-	EOF
-	chmod +x attrdump.sh &&
-	start_broker conf8 fake0 ./attrdump.sh >ipc.out &&
-        F0=$! &&
-	start_broker conf8 fake1 &&
-        F1=$! &&
-	wait $F0 $F1 &&
-	echo 2 >ipc.exp &&
-	echo "fake[0-1]" >> ipc.exp &&
+	flux start -s2 --test-hosts=fake[0-1] \
+		-o,-Sbroker.rc1_path=,-Sbroker.rc3_path= \
```

Comment:
could use ${ARGS}? (also below)

Reply:
`${ARGS}` doesn't quite work there because its not "wrapped" for `flux-start -o` e.g.
```
ARGS="-Sbroker.rc1_path= -Sbroker.rc3_path="
```
vs
```
"-o,-Sbroker.rc1_path=,-Sbroker.rc3_path="
```
It didn't seem worth it to me to create another variable or post-process.

BTW, I'll see if I can fill in a bit of missing coverage for the new `flux-start` option tomorrow and get this into the green.